### PR TITLE
ref: Add ruff rules for arguments (ARG)

### PR DESCRIPTION
**Pull request by cbornet**
Add ruff rules for arguments (ARG)

### CODE REVIEW

File: src/backend/base/langflow/__main__.py [hunk 1]
```diff
@@ -97,12 +97,12 @@ def run(
     ),
     log_level: str | None = typer.Option(None, help="Logging level.", show_default=False),
     log_file: Path | None = typer.Option(None, help="Path to the log file.", show_default=False),
-    cache: str | None = typer.Option(
+    cache: str | None = typer.Option(  # noqa: ARG001
```
Comment (cbornet) [inline src/backend/base/langflow/__main__.py:100]: Used by reflection (inspect.getargvalues)

File: src/backend/base/langflow/worker.py [hunk 2]
```diff
@@ -31,10 +31,10 @@ def build_vertex(self, vertex: Vertex) -> Vertex:
 
 @celery_app.task(acks_late=True)
 def process_graph_cached_task(
-    data_graph: dict[str, Any],
-    inputs: dict | list[dict] | None = None,
-    clear_cache=False,
-    session_id=None,
+    data_graph: dict[str, Any],  # noqa: ARG001
```
Comment (cbornet) [inline src/backend/base/langflow/worker.py:34]: Not sure what the impact would be for celery to rmove these args

File: src/backend/base/langflow/graph/vertex/base.py [hunk 3]
```diff
@@ -635,7 +635,11 @@ def _log_transaction_async(
         self.log_transaction_tasks.add(task)
         task.add_done_callback(self.log_transaction_tasks.discard)
 
-    async def _get_result(self, requester: Vertex, target_handle_name: str | None = None) -> Any:
+    async def _get_result(
+        self,
+        requester: Vertex,
+        target_handle_name: str | None = None,  # noqa: ARG002
```
Comment (cbornet) [inline src/backend/base/langflow/graph/vertex/base.py:641]: Used in subclasses

File: src/backend/base/langflow/worker.py (hunk 2 above)
Comment (ogabrielluiz) [reply src/backend/base/langflow/worker.py:34]: celery is not supported atm (i.e we need to rebuild the integration)

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Pull request merged**